src/examples/react-starter/Viz.js

```diff
@@ -0,0 +1,83 @@
+import React, { useEffect } from 'react';
+import * as d3 from 'd3';
```

Comment:
If you grab d3 version 4+ you can probably just import d3-select or whatever you need since d3 is a very big lib.